docs/dashboard/apps.md

```diff
@@ -64,7 +64,17 @@ To learn more about the webhook protocols, see the [Developer's Guide](developer
 
 ### Events
 
-In the _Events_ card, you can assign singular events that you want to be triggered by the webhook. Alternatively, you can choose the _All events_ option to send notifications about all events, regardless of type. By choosing this option you will send payloads for all events currently assigned, in doing so your range will change as events are added or deleted by the Saleor team. If you want to assign a limited range of events, choose them from the menu of specific items.
+:::note
+Asynchronous and synchronous webhook events are available in Saleor 3.1 (alpha stage).
+:::
+
+In the *Events* card, you can assign singular events that you want to be triggered by the webhook.
+
+There are two types of events that can be assigned to a webhook: [synchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/synchronous-webhooks) and [asynchronous](https://docs.saleor.io/docs/3.0/developer/extending/apps/asynchronous-webhooks). 
+
+Payment webhooks are synchronous webhooks that allow delegating payment handling to Saleor Apps. You can select multiple synchronous events.
```

Comment:
Maybe I don't understand it well, but I assumed after reading this line, that synchronous webhooks means payments webhooks. (Maybe we could add a single line about sync webhooks similar to line for Asynchronous webhook)
Payment webhooks are a part of synchronous webhooks. A synchronous webhook can be a payment webhook but it also can be something different.